src/RadioBoards.h

```diff
@@ -77,12 +82,15 @@
 #elif defined(RADIO_BOARD_RASPBERRYPI_PICO)
   #include "contributed/RaspberryPi/PI_PICO.h"
 
+#elif defined(RADIO_BOARD_FEATHER_RP2040)
+  #include "contributed/Adafruit/Feather_RP2040.h"
+
 #else
   #error "Unsupported or unknown radio board!"
 
 #endif
 
-#if RADIO_BOARDS_DEBUG
+#if defined (RADIO_BOARDS_DEBUG)
```

Comment:
I propose to keep it as it was - the intention (admittedly undocumented) was to have the macros decide on 0/1, same as `RADIOLIB_DEBUG`.